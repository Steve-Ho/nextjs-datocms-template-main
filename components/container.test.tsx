import {expect, test} from '@jest/globals';
import {create} from 'react-test-renderer';
import Container from './container';

test('snapshot', () => {
	const result = create(
		<Container>
			<p>Hello World</p>
		</Container>,
	).toJSON();

	expect(result).toMatchSnapshot();
});
