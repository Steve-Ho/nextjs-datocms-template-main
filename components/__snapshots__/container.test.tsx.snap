// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="container"
>
  <p>
    Hello World
  </p>
</div>
`;
